packages/contracts-bedrock/justfile

```diff
@@ -59,7 +59,7 @@ snapshots-abi-storage:
 
 snapshots: build snapshots-no-build
 
-snapshots-no-build: snapshots-abi-storage kontrol-summary-fp kontrol-summary
+snapshots-no-build: semver-lock snapshots-abi-storage kontrol-summary-fp kontrol-summary
```

Comment:
Should `semver-lock` then be removed from `pre-pr-no-build`?

Reply:
I am not sure what workflows people use to be honest. I just usually do `just semver-lock` then `just snapshots`. Curious what @mds1 thinks

Reply:
Ah, so I didn't realize this since I haven't used `pre-pr-no-build`, but that already does the `semver-lock` check. 

The full set of options is:

```just
pre-pr-no-build: gas-snapshot-no-build snapshots-no-build semver-lock autogen-invariant-docs lint
pre-pr: clean build-go-ffi build pre-pr-no-build
pre-pr-full: test validate-deploy-configs validate-spacers pre-pr
```

So we probably can close this PR, and, as already mentioned in CONTRIBUTING.md tell contributors to use `just pre-pr` or `just pre-pr-full`?